examples/urlInterpolationDemo.html

```diff
@@ -83,8 +84,12 @@ <h2>Localization Demo</h2>
                         pcEnv = getParamValue(currParam);
                     } else if(currParam[0] === 'environment' && pcEnv === null) {
                         pcEnv = getParamValue(currParam);
+                    } else if(currParam[0] === 'conversationId') {
```

Comment:
I'm assuming my thought process was to use a different queryParamName than the replacement token to show the variable could be different.  But, in hind sight, maybe that's confusing?

Thoughts about supporting both param names for pcLangTag and pcConversationId?  For instance:
`['langTag', 'pcLangTag'].indexOf(paramName) === 0` and
`['conversationId', 'pcConversationId'].indexOf(paramName) === 0`?

Reply:
Yeah I see what you mean. That's a good idea, I'll add that.

Reply:
Your syntax still only includes the first index, should it be `!== -1` instead of `=== 0`?